src/main/interface/settings.c

```diff
@@ -263,6 +263,18 @@ static const char * const lookupTableLowpassType[] = {
     "FAST_KALMAN"
 };
 
+static const char * const lookupTableDtermLowpassType[] = {
+    "PT1",
+    "BIQUAD",
+#if defined(USE_FIR_FILTER_DENOISE)
+    "FIR",
+#endif
+    NULL,
```

Comment:
I don't think this works :(

We need to use:

```
#else
   NULL,
#endif
```

and also, the functionality for NULL entry in LUT does not exist. @mikeller was going to add?

Reply:
disregard previous